etc/ReplayOfflineConfiguration.py

```diff
@@ -124,9 +127,9 @@
 alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "113X_dataRun3_Express_Candidate_2021_07_02_14_38_40"
-promptrecoGlobalTag = "113X_dataRun3_Prompt_v3"
-alcap0GlobalTag = "113X_dataRun3_Prompt_v3"
+expressGlobalTag = "120X_dataRun3_Express_Candidate_2021_09_30_18_36_13"
+promptrecoGlobalTag = "120X_dataRun3_Prompt_Candidate_2021_10_01_08_44_53"
+alcap0GlobalTag = "120X_dataRun3_Prompt_Candidate_2021_10_01_08_44_53"
```

Comment:
Hi @germanfgv please note that in the gDoc (https://docs.google.com/document/d/1dfxw7EE7Bl66QwXgnYC60_XixX0cUTTv-weAB4hdYDE/edit#) the GTs for running the replay on run2 data are different wrt to what you put here:
- Prompt : 120X_dataRun3_Prompt_Candidate_2021_09_30_19_06_33
- Express: 120X_dataRun3_Express_Candidate_2021_09_30_18_52_55

Reply:
Thanks for the correction Francesco. I changed the GT yesterday  tried again. The replay is almost done, with DoubleMuon workflows already completed.
I'm waiting for the last few SingleMuon jobs to finish before reporting the results.